django-ory-cloud/Makefile

```
@@ -0,0 +1,20 @@
+install: node_modules  poetry.lock
+	poetry install
+
+# .PHONY: build-examples
+# build-examples:
+# 		cd code-examples/protect-page-login/nextjs && npm run build
+# 		cd code-examples/protect-page-login/flutter_web_redirect && flutter build web --web-renderer html
+# 		cd code-examples/protect-page-login/vue && VUE_APP_API_URL=http://localhost:4007 VUE_APP_ORY_URL=http://localhost:3006 npm run build
+# 		cd code-examples/protect-page-login/react && npm run build
+
+.PHONY: test
+test: install .bin/ory
+	./test.sh
+
+.bin/ory: Makefile
+	curl https://raw.githubusercontent.com/ory/meta/master/install.sh | bash -s -- -d -b .bin ory v0.1.33
```

The installer script doesn't contain any bashisms ([proof](https://github.com/ory/meta/blob/master/install.sh#L1), so we can get rid of the Bash dependency altogether here (woohoo!)

Also updating to the latest version of Ory CLI.

```suggestion
.bin/ory: Makefile
	curl https://raw.githubusercontent.com/ory/meta/master/install.sh | sh -s -- -d -b .bin ory v0.1.36
```